### ISSUE TITLE
Strange compile problem

### ISSUE THREAD

**Issue by bibhas2** [retep998/winapi-rs#245]
I am able to call most ``gdi32`` functions fine until I tried to use ``CreatePen()``:

```
error: unresolved name `gdi32::CreatePen`
```

My code:

```
//Does not compile
let pen = gdi32::CreatePen(winapi::PS_SOLID, 5, 0);

//Compiles fine
gdi32::DeleteObject(pen);
```

Relevant ``Cargo.lock`` section:

```
dependencies = [
 "gdi32-sys 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "kernel32-sys 0.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
 "libc 0.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "user32-sys 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
 "winapi 0.2.5 (registry+https://github.com/rust-lang/crates.io-index)",
]
```

Strange and can't find out why.

**Comment by retep998** [comment]
I'm pretty sure this is just because bindings to `CreatePen` were added after the last version of `gdi32-sys` was published. This will be resolved once I publish a new version of `gdi32-sys` at which point you just have to update to the new version.

**Comment by Eh2406** [comment]
Did this get fixed with "0.2.0" if so can we close?

**Comment by bibhas2** [comment]
I will not get a chance to validate. Please close. Thank you.
